include/heap.h

```diff
@@ -1,9 +1,37 @@
 #ifndef POKEDIAMOND_HEAP_H
 #define POKEDIAMOND_HEAP_H
 
-void * AllocFromHeap(u32 heap_id, u32 size);
-void * AllocFromHeapAtEnd(u32 heap_id, u32 size);
-void FreeToHeap(void * ptr);
+#include "global.h"
+
+struct UnkStruct_020166C8
```

Comment:
this isn't normally how struct members are named, unk00 for item at 0x00 etc etc